src/core_plugins/timelion/server/lib/classes/datasource.js

```diff
@@ -40,7 +40,7 @@ export default class Datasource extends TimelionFunction {
     config.args.push({
       name: 'offset',
       types: ['string', 'null'],
-      help: 'Offset the series retrieval by a date expression. Eg -1M to make events from one month ago appear as if they are happening now'
+      help: 'Offset the series retrieval by a date expression. E.g., -1M to make events from one month ago appear as if they are happening now'
```

Comment:
https://kibana-ci.elastic.co/job/elastic+kibana+pull-request+multijob-intake/5104/console
17:26:43   43:1  error  Line 43 exceeds the maximum line length of 140  max-len